spago.yaml

```diff
@@ -0,0 +1,17 @@
+workspace:
+  packageSet:
+    registry: 50.7.0
+  extraPackages: {}
+
+package:
+  name: spec-discovery
+  dependencies:
```

Comment:
Do the dependencies have to be explicitly versioned? I see the weren't versioned in the old spago config.

Reply:
No, they weren't, but I think it's a good practice to specify version numbers nevertheless.

In Haskell you have things like: https://hackage.haskell.org/package/cabal-constraints which pin the versions exactly.

If spec releases version 8.x.x, someone will soon discover that `purescript-spec-discovery` won't compile because of dependency errors and would try to fix it to work with newer version.

Reply:
Isn't the whole point of package sets to avoid these things? 

For example, if there is a new version of spec that makes discovery not build anymore, then discovery won't be included in the next package set. And anybody who needs it can just stay with the previous package set version, not upgrade to the new one.

Reply:
Yeah, maybe you're right, I'll fix these.

Reply:
Ok, good for now. I'll look into this some more later.